### PR TITLE
Semver auto deploy schedules

### PR DESCRIPTION
**Pull request by sgalsaleh**

### CODE REVIEW

File: pkg/autodeployer/autodeployer.go [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+func Start() error {
+	logger.Debug("starting auto deployer")
+
+	appsList, err := store.GetStore().ListInstalledApps()
+	if err != nil {
+		return errors.Wrap(err, "failed to list installed apps")
+	}
+
+	for _, a := range appsList {
+		if a.IsAirgap {
```
Comment (divolgin) [inline pkg/autodeployer/autodeployer.go:34]: `Configure` also checks this

Comment (sgalsaleh) [reply pkg/autodeployer/autodeployer.go:34]: which is fine, can't be too careful :). configure is called by other packages as well

File: pkg/autodeployer/autodeployer.go [hunk 2]
```diff
@@ -0,0 +1,262 @@
...
+	if err != nil {
+		return -1
+	}
+
+	pendingVersions := PendingVersions{}
+
+	for _, v := range versions {
+		switch opts.SemverAutoDeploy {
+		case apptypes.SemverAutoDeployPatch:
+			s, err := semver.ParseTolerant(v.VersionLabel)
```
Comment (divolgin) [inline pkg/autodeployer/autodeployer.go:219]: there should be `v.Semver` already parsed.

File: pkg/autodeployer/autodeployer.go (hunk 1 above)
Comment (sgalsaleh) [reply pkg/autodeployer/autodeployer.go:34]: i can get rid of it

**Review by divolgin — APPROVED** [review]

**Review by divolgin — APPROVED** [review]

**Pull request merged**